samples/Mimban/Mimban.Server/Program.cs

```diff
@@ -126,7 +126,7 @@ await using (var scope = app.Services.CreateAsyncScope())
     var identity = new ClaimsIdentity(
         authenticationType: TokenValidationParameters.DefaultAuthenticationType,
         nameType: Claims.Name,
-        roleType: Claims.Role);
+        roleType: Claims.Role);s
```

Comment:
You probably don't want an S here.

Reply:
Hehe, I deliberately added it (see https://github.com/openiddict/openiddict-samples/pull/216/commits/3f421c37c74f5382846dbfc82520e68ee8ba9b6b) to confirm that the build turned red on a compilation error with the fix in place (without this fix, the build stays green, which is not... ideal 😄)